Release notes for 11-09-2023 console release (#1184)
neondatabase/website: content/release-notes/2023-11-10-console.md

### Fixes & improvements

- UI: Added Neon serverless driver HTTP and WebSockets connection examples to the **Connection Details** widget on the Neon **Dashboard**.
- UI: Updated the Django code snippet in the **Connection Details** widget to set `DISABLE_SERVER_SIDE_CURSORS` to `true` when using a pooled connection.

> - UI: Fixed an issue that prevented email-authenticated Neon users from logging in via the Neon CLI (`neonctl auth`).

Reviewer: this affected all of the Oauth use cases, so I would suggest something like this

```suggestion
- UI: Fixed an issue that prevented email-authenticated Neon users from authorizing an OAuth access.
```

Author: Fixed. Thank you.
